controllers/authentication/openidconnect_controller.go

```diff
@@ -35,6 +35,8 @@ type OpenIDConnectReconciler struct {
 	*unionAuthTokenHandler
 }
 
+var ResyncPeriod time.Duration
```

Comment:
Don't expose this as global variable - add this to the controller struct and have `AddFlags` and `ApplyTo` functions that can be used to add the flags and apply the value of those flags to the configuration